### DISCUSSION TITLE
Release to unraid?

### DISCUSSION THREAD

**Discussion by ChadTaljaardt** [bubuntux/nordlynx#4] [Ideas]
Hello, 

I use the other repo for nordvpn but noticed you suggesting others to switch to this repo. Can you release this repo on unraid so we can switch?

Thanks!

**Comment by bubuntux** [comment]
not sure what you mean by `release` to unraid, can you clarify

**Reply by ChadTaljaardt** [reply]
I tried to find this application in the "community apps" on unraid. 

I am fairly new to unraid and dont know how those apps get on there, but i am currently using on which seems to be made by you on there, and i cannot find this repo on there. 

![image](https://user-images.githubusercontent.com/13891249/142091741-0bcb8954-c9ba-4ef4-b8fb-e6bca937a2a3.png)

**Reply by bubuntux** [reply]
sry but this project is a docker image, search how to run docker images on unraid

**Comment by ETA-22** [comment]
> sry but this project is a docker image, search how to run docker images on unraid

Yeah, unraid has Docker images ready to go, like your NordVPN. I’m assuming there’s a template or something that needs to be included, then it’s submitted to the unraid apps (Docker) DB. 

A lot of your users are still using the NordVPN docker in unraid (looks like a huge amount of your user base going by the app installers stats) because NordLynx isn’t readily available there.

So, if you’d like people to migrate to NordLynx, that would sure help. :)

**Reply by ChadTaljaardt** [reply]
I want to make the switch over, but lack of that application being readily available is stopping me. 

I am looking into making the template for this application but just need to find the time to get it working.

**Reply by ChadTaljaardt** [reply]
Ive been following this guide: 

https://forums.unraid.net/topic/57181-docker-faq/#comment-566084

https://www.youtube.com/watch?v=g6iMwea-HLY
